LuaSkin/LuaSkin/Skin.h

```diff
@@ -45,6 +45,9 @@ extern int luaopen_luaskin_internal(lua_State* L) ; // entry vector to luaskin.m
 // Define a break variable for the reference checker
 #define LS_RBREAK INT_MIN
 
+// Define a hacky fixed UUID refTable value to indicate LUA_REGISTRYINDEX
+#define LS_REGISTRYINDEX_REF @"11111111-1111-1111-1111-111111111111"
```

Comment:
I hate this, but I don't currently have a better idea.

Reply:
Well, I kinda do, and that would be to subclass NSUUID. Kinda wish instead of going through all these files and changing the type of refTable to NSUUID, I'd set up a typedef. I may well go back and do that.